proposals/lip-0061.md

```diff
@@ -405,7 +405,7 @@ The height up to which single commits can be removed.
 ```python
 def getMaxRemovalHeight() -> uint32:
     b = block at height maxHeightFinalized
-    return max(b.header.aggregateCommit.height, genesisHeight)
+    return max(b.header.aggregateCommit.height, MIN_CERTIFICATE_HEIGHT - 1)
 ```
 
 In the function above, `maxHeightFinalized` is the maximum height up to which the current chain is considered final, as introduced in [LIP 0056](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0056.md#finalized-height), and `genesisHeight` is the height of the genesis block.
```

Comment:
Comment about line 415: We implicitly assume that `h2 > = h1`. This was true in the previous setting but not anymore. We could just say that if `h1 > h2` we just don't create single commits at all I guess..

Reply:
> and `genesisHeight` is the height of the genesis block.

This part should be removed or replaced by a comment for `MIN_CERTIFICATE_HEIGHT`

Reply:
Good point. I guess it should be removed, since `MIN_CERTIFICATE_HEIGHT` is well-defined

Reply:
- I removed the part "and genesisHeight is the height of the genesis block.".
- I added the requirement that `h2 > h1` (note that this is different to what you proposed @gkoumout )
- I did not swap the sections to keep the diff simple and to follow the philosophy now to only change what is strictly required.

Reply:
Can be resolved

Reply:
@janhack Could you clarify the difference between what I proposed and what you added? I fail to see it. 

Also, swapping of the sections was proposed to avoid mis-conceptions of the paragraph, since you essentially run the same "algorithm" for a different interval [h1,h2], so it would be easier to parse if we first see the description and then the modification. At least I got confused while reviewing the implementation due to this. 

But ok, since it is ok now, I agree it is an optional change.

Reply:
You proposed `h2 > = h1` or (alternatively `not h1 > h2`), while I now wrote `h2 > h1`, i.e., strict inequality.

As mentioned above, I would keep the order of sections as is.